lib/connection.cpp

```diff
@@ -2485,3 +2486,8 @@ bool Connection::isVerifiedSession(const QString& megolmSessionId) const
     return query.next() && query.value("verified").toBool();
 }
 #endif
+
+void Connection::completeSetup(const QString &mxId)
+{
+    d->completeSetup(mxId);
+}
```

Comment:
Given that you always create a `Connection` object just before calling this wrapper, it makes sense to have a static factory instead:
```suggestion
Connection* Connection::makeMockConnection(const QString& mxId)
{
    auto* c = new Connection;
    c->d->completeSetup(mxId);
    return c;
}
```